.github/workflows/ci.yml

```diff
@@ -174,7 +174,7 @@ jobs:
           save-if: ${{ github.event_name != 'merge_group' }}
 
       - name: cargo nextest
-        run: cargo nextest run --workspace --locked --all-features --no-fail-fast
+        run: cargo nextest run --workspace --locked --all-features --no-fail-fast --no-tests=pass
```

Comment:
I'd rather find a test to add, but this is also a reasonable workaround.

Reply:
Yeah, later this week.